config/services-prod.yml

```diff
@@ -600,6 +600,40 @@ https://cmr.earthdata.nasa.gov:
         is_sequential: true
       - image: !Env ${NET2COG_IMAGE}
 
+  - name: giovanni-averaging-service
+    description: |
+      A service to compute averaging over specified dimensions on Zarr data on a Dask cluster running in Cloud Giovanni. A time-averaged map
+      will produce a GEOTIFF and an area-averaged timeseries will produce a CSV.
+    data_operation_version: '0.20.0'
```

Comment:
Note we should update this to the latest version `0.22.0` in both UAT and prod soon. It should be backwards compatible but I wouldn't update it to the latest until you've tested in UAT first.